include/cvc5/cvc5_skolem_id.h

```diff
@@ -654,6 +654,50 @@ enum ENUM(SkolemId) : uint32_t
    * - Type: ``E``
    */
   EVALUE(SETS_MAP_DOWN_ELEMENT),
+  /**
+   * A skolem function that is unique per floating-point sort, introduced for
+   * the undefined zero case of ``fp.min``.
+   *
+   * - Number of skolem indices: ``1``
+   *   - ``1:`` The Sort of the fp.min operator.
```

Comment:
Document the types of the skolems as well.

Reply:
Why do we document this as `Type` when this is also visible on the API level?